tests/e2e/specs/product-feed/product-feed-campaign-notice.test.js

```diff
@@ -0,0 +1,207 @@
+	test.beforeAll( async ( { browser } ) => {
+		page = await browser.newPage();
+		productFeedPage = new ProductFeedPage( page );
+		setupAdsAccounts = new SetupAdsAccountsPage( page );
+		await Promise.all( [
+			setupAdsAccounts.mockAdsAccountsResponse( [] ),
+			productFeedPage.mockRequests(),
+			setOnboardedMerchant(),
+		] );
+	} );
+
+	test.afterAll( async () => {
+		await clearOnboardedMerchant();
+		await page.close();
+	} );
+
+	test.describe( 'No campaign', () => {
+		test.beforeAll( async () => {
+			await productFeedPage.fulfillAdsCampaignsRequest( [] );
+			await productFeedPage.goto();
+		} );
+
+		test( 'No active product and no campaign; Do not display campaign notice', async () => {
+			await expect(
+				page.getByRole( 'heading', { level: 1, name: 'Product Feed' } )
+			).toBeVisible();
+
+			await productFeedPage.fulfillProductStatisticsRequest( {
+				timestamp: 1695011644,
+				statistics: {
+					active: 0,
+					expiring: 0,
+					pending: 0,
+					disapproved: 0,
+					not_synced: 1137,
+				},
+				scheduled_sync: 0,
+				loading: false,
+			} );
+
+			await page.reload();
+			await page.waitForLoadState( LOAD_STATE.DOM_CONTENT_LOADED );
+
+			await expect(
+				page.getByRole( 'heading', {
+					name: 'Overview',
+				} )
+			).toBeVisible();
+
+			await expect(
+				await productFeedPage.getActiveProductValueElement()
+			).toBeVisible();
+
+			await expect(
+				( await productFeedPage.getActiveProductValue() ).trim()
+			).toBe( '0' );
+
+			await expect(
+				await productFeedPage.getCampaignNoticeSection()
+			).not.toBeVisible();
+		} );
+
+		test( 'Has active product but no campaign; Display campaign notice', async () => {
+			await productFeedPage.fulfillProductStatisticsRequest( {
+				timestamp: 1695011644,
+				statistics: {
+					active: 1,
+					expiring: 0,
+					pending: 0,
+					disapproved: 0,
+					not_synced: 1137,
+				},
+				scheduled_sync: 0,
+				loading: false,
+			} );
+
+			await page.reload();
+			await page.waitForLoadState( LOAD_STATE.DOM_CONTENT_LOADED );
+
+			await expect(
+				await productFeedPage.getActiveProductValueElement()
+			).toBeVisible();
+
+			await expect(
+				( await productFeedPage.getActiveProductValue() ).trim()
+			).toBe( '1' );
+
+			const noticeSection =
+				await productFeedPage.getCampaignNoticeSection();
+			const createCampaignButton =
+				await productFeedPage.getInNoticeCreateCampaignButton();
+
+			await expect( noticeSection ).toBeVisible();
+			await expect( createCampaignButton ).toBeVisible();
+			await createCampaignButton.click();
+			await page.waitForLoadState( LOAD_STATE.DOM_CONTENT_LOADED );
+			await expect(
+				page.getByRole( 'heading', {
+					level: 1,
+					name: 'Set up your accounts',
+				} )
+			).toBeVisible();
+
+			await productFeedPage.goto();
+			await page.waitForLoadState( LOAD_STATE.DOM_CONTENT_LOADED );
+
+			await expect(
+				page.getByRole( 'heading', { level: 1, name: 'Product Feed' } )
+			).toBeVisible();
```

Comment:
Why does it need to go back here?

Reply:
You are right; we can skip this.

Reply:
The page eventually crashed for some reason; all I needed was to check if `Create Campaign` was taking me to the right page and not to test the entire functionality on that page. So, going back was a good option here.